src/main/resources/project/ec_setup.pl

```diff
@@ -86,6 +86,12 @@
     description => "Puppet version",
     category    => "Resource Management"
 );
+my %PuppetAgentSetConfig = (
+    label       => "Puppet - AgentSetConfig",
```

Comment:
Right  .But we already have configure agent procedure, so could not think of a better name. We can use this name.  Should we consider renaming ConfigureAgent also as now with configurations in picture this name can be misleading.